runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -363,8 +363,8 @@ else if (message.equals(RING_OF_FORGING_BREAK_TEXT))
 				{
 					notifier.notify("Your ring of forging has melted.");
 				}
-
-				updateRingOfForgingCharges(MAX_RING_OF_FORGING_CHARGES);
+				//Once your ring melts while smelting ore, the RING_OF_FORGING_USED_TEXT is still printed to the chatbox, which subtracts 1 from the counter
```

Comment:
it would be more accurate to say that the equipment isnt cleared before reading the use message after the break, so it still thinks to deduct one charge from the already reset counter, so a fake charge is added here to counteract that